insights/parsers/journal_all.py

```diff
@@ -0,0 +1,44 @@
+"""
+JournalAll file ``/sos_commands/logs/journalctl_--no-pager``
+=========================================================================
+"""
+
+from .. import Syslog, parser
+from insights.specs import Specs
+
+
+@parser(Specs.journal_all)
+class JournalAll(Syslog):
+    """
+    Read the ``/sos_commands/logs/journalctl_--no-pager`` file.  Uses the
+    ``Syslog`` class parser functionality - see the base class for more details.
+
+    Sample log lines::
+
+        -- Logs begin at Wed 2017-02-08 15:18:00 CET, end at Tue 2017-09-19 09:25:27 CEST. --
+        May 18 15:13:34 lxc-rhel68-sat56 jabberd/sm[11057]: session started: jid=rhn-dispatcher-sat@lxc-rhel6-sat56.redhat.com/superclient
+        May 18 15:13:36 lxc-rhel68-sat56 wrapper[11375]: --> Wrapper Started as Daemon
+        May 18 15:13:36 lxc-rhel68-sat56 wrapper[11375]: Launching a JVM...
+        May 18 15:24:28 lxc-rhel68-sat56 yum[11597]: Installed: lynx-2.8.6-27.el6.x86_64
+        May 18 15:36:19 lxc-rhel68-sat56 yum[11954]: Updated: sos-3.2-40.el6.noarch
+
+    Note:
```

Comment:
This format of "Note" is a good approach (in the original `journal_since_boot`), would you please modify this one (and only this one, doesn't mean the one in `journal_since_boot`) according to the `Sphinx` style?

See:  https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

Reply:
Thanks! Updated.